config/initializers/mime_types.rb

```diff
@@ -1,8 +1,7 @@
-# Be sure to restart your server when you modify this file.
-
-# Add new mime types for use in respond_to blocks:
-# Mime::Type.register "text/richtext", :rtf
-# Mime::Type.register_alias "text/html", :iphone
-
-Mime::Type.register "application/vnd.safecast+json;version=1.0", :safecast_api_v1_json
-Mime::Type.register "application/vnd.safecast+xml;version=1.0",  :safecast_api_v1_xml
+{
+  safecast_api_v1_json: 'application/vnd.safecast+json;version=1.0',
+  safecast_api_v1_xml: 'application/vnd.safecast+xml;version=1.0',
+  kml: 'application/vnd.google-earth.kml+xml'
+}.each do |sym, str|
```

Comment:
Feels a little YAGNI here. Do we have plans for adding many more mime types you think?